proto/bff/v1/moderation_service.proto

```diff
@@ -97,4 +98,26 @@ message ProcessApprovalQueueRequest {
   bool is_artist = 3;
 }
 
-message ProcessApprovalQueueResponse {}
\ No newline at end of file
+message ProcessApprovalQueueResponse {}
+
+message AuditEntry {
```

Comment:
**suggestion**: we could add a field for the type of action that this entry describes, similar to how [GitLab (and likely GitHub) handles system notes](https://gitlab.com/gitlab-org/gitlab/-/blob/6c3d58720f4fda963a5636a11c65e15662ca354c/app/models/system_note_metadata.rb#L23), which are immutable and analog to the use case of our moderation log. An entry with this type set to `null` would be a regular text comment. In all other cases, such as rejecting a pending user or removing them from the feed, the `comment` could be used for an optional reason.